### ISSUE TITLE
Cannot read property 'members' of undefined for NgxMatColorPickerModule

### ISSUE THREAD

**Issue by pkojchevski** [h2qutc/angular-material-components#68]
ERROR in Cannot read property 'members' of undefined for NgxMatColorPickerModule.

**Comment by wmehanna** [comment]
I got this issue after upgrading to angular 10.  It seems that I had entryComponent left in my modules.  I got rid of the and then no more of this error.

**Comment by h2qutc** [comment]
@wmehanna  @pkojchevski  If you use Angular 10, can you check the version v4.0.0 out ? 
I think that this bug is fixed in latest version
```
npm install --save  @angular-material-components/color-picker@4.0.0
```

**Comment by wmehanna** [comment]
> @wmehanna @pkojchevski If you use Angular 10, can you check the version v4.0.0 out ?
> I think that this bug is fixed in latest version
> 
> ```
> npm install --save  @angular-material-components/color-picker@4.0.0
> ```

thank for the info!  You just saved me few hours of work.

**Comment by toarnold** [comment]
I have this error using Angular 8. I'm not able to upgrad to Angular 10 at th emoment. Is there a possible work around?
If I try to get a production uild. The error is more verbose:
`ERROR in Unexpected value 'undefined' exported by the module 'NgxMatColorPickerModule in D:/MyApp/node_modules/@angular-material-components/color-picker/angular-material-components-color-picker.d.ts'`

**Comment by pkojchevski** [comment]
i also was not able to upgrade to 10...

**Comment by manvithareddy17** [comment]
Even I am having a similar issue.  I am using Angular 7
ng build- cannot read property 'members' of undefined
ng build prod- ERROR in Unexpected value 'undefined' exported by the module 'NgxMatColorPickerModule in D:/MyApp/node_modules/@angular-material-components/color-picker/angular-material-components-color-picker.d.ts'

Please help. Thanks.

**Comment by manvithareddy17** [comment]
Even I am having a similar issue.  I am using Angular 7
ng build- cannot read property 'members' of undefined
ng build prod- ERROR in Unexpected value 'undefined' exported by the module 'NgxMatColorPickerModule in ....(My Project Path).../node_modules/@angular-material-components/color-picker/angular-material-components-color-picker.d.ts'

Please help. Thanks.

**Comment by h2qutc** [comment]
@toarnold Did you try the version v4.0.0 that supports Angular 10 ?
@manvithareddy17  Can you let me know which version of @angular-material-components/color-picker that you used plz?

**Comment by pkojchevski** [comment]
i can not upgrade the project to angular10

**Comment by manvithareddy17** [comment]
Yes. I have tried updating to 4.0.0! But got the similar issue. Tried
deleting and reinstalling as well. But no luck.

Thanks,
Manvitha

**Comment by toarnold** [comment]
> Did you try the version v4.0.0 that supports Angular 10 ?

Yes, I tried ALL awailable versions, but the result is the same.

**Comment by toarnold** [comment]
I have build a small repro scenario. `ng server` and `ng build` leads to
`ERROR in Cannot read property 'members' of undefined`
[ColorPickerDemo.zip](https://github.com/h2qutc/angular-material-components/files/5026361/ColorPickerDemo.zip)

**Comment by AndreyChayko** [comment]
I guess nobody plan to look into this.. So guys, I have to spend 3 hours to reimplement the color-picker functionality using [this fantasctic plugin](https://www.npmjs.com/package/ngx-color) and I can suggest all of you t odo the same

**Comment by digital195** [comment]
Are there any updates on this error? I'm on angular version 7 with color picker version 2.0.2. Still getting that error. On ng serve after a file was changed and it was triggered to recompile the changes the import is working and the error is gone. Somebody knows a solution?

```Date: 2021-02-02T17:09:47.209Z
Hash: 7aef861966dabda45424
Time: 10887ms
chunk {main} main.js, main.js.map (main) 1.9 kB [initial] [rendered]
chunk {polyfills} polyfills.js, polyfills.js.map (polyfills) 93.2 kB [initial] [rendered]
chunk {runtime} runtime.js, runtime.js.map (runtime) 6.08 kB [entry] [rendered]
chunk {scripts} scripts.js, scripts.js.map (scripts) 194 kB [entry] [rendered]
chunk {styles} styles.js, styles.js.map (styles) 727 kB [initial] [rendered]
chunk {vendor} vendor.js, vendor.js.map (vendor) 328 kB [initial] [rendered]

ERROR in Cannot read property 'members' of undefined
i ｢wdm｣: Failed to compile.```

**Comment by mehul-warhousing1** [comment]
any updates on this ?

**Comment by mehul-warhousing1** [comment]
guys, any update on this ?
